test/integration/targets/azure_rm_aci/aliases

```diff
@@ -0,0 +1,2 @@
+cloud/azure
+destructive
```

Comment:
Add the `posix/ci/cloud/group2/azure` alias so the tests can run as part of CI.

Reply:
Thanks Matt! I wasn't aware it's needed to run CI. Now actually my test is running... and failing...

"Error creating the ACI instance: Azure Error: MissingSubscriptionRegistration\nMessage: The subscription is not registered to use namespace 'Microsoft.ContainerInstance'

I guess this has to be somehow fixed in the subscription used by CI...

Reply:
I've registered `Microsoft.ContainerInstance`, so the tests should be able to run now.

Reply:
@mattclay wanted to check have you also registered container register? Reference: #32481

Reply:
I've registered `Microsoft.ContainerRegistry` as well.